test/test_helpers.py

```diff
@@ -22,6 +22,8 @@
     'test.vtk',
     'test.vtu',
     'test.xmf',
+    'test.1.off',
```

Comment:
I'm not sure if you want to merge it like this or if I should add an example for all variants or a dedicated test?